components/centraldashboard/cypress/e2e/landing-page.cy.ts

```diff
@@ -16,8 +16,8 @@ describe('Landing Page', () => {
   it('should access the landing page', ()=>{
     cy.get('main-page').shadow().find('landing-page').should('exist');
     cy.get('main-page').shadow().find('landing-page').shadow().find('#MainCard > neon-animatable > h2').should('have.text', 'Welcome');
-    cy.get('main-page').shadow().find('landing-page').shadow().find('#MainCard > neon-animatable > aside').find('a').should('exist').and('have.prop', 'href', 'https://business-transformation.pages.cloud.statcan.ca/aaw/aaw/en/');
-    cy.get('main-page').shadow().find('landing-page').shadow().find('#MainCard > neon-animatable').find('md2-input#Email').should('exist').and('have.prop', 'value', 'user.name@email.com');
+    cy.get('main-page').shadow().find('landing-page').shadow().find('#MainCard > neon-animatable > aside').find('a').should('exist').and('have.prop', 'href', 'https://business-transformation.pages.cloud.statcan.ca/zone/aaw/en/');
```

Comment:
This is the wrong link https://business-transformation.pages.cloud.statcan.ca/zone/aaw/en/
It should be https://business-transformation.pages.cloud.statcan.ca/aaw/zone/en/

Reply:
should be fixed